EnglishFilter/sections/general_extensions.txt

```diff
@@ -284,6 +284,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 ! popup on convert
 yout.pw#%#//scriptlet("prevent-window-open")
+! https://github.com/AdguardTeam/AdguardFilters/issues/122003
+javhd.icu#%#//scriptlet('prevent-window-open')
```

Comment:
This scriptlet may be used
`#%#//scriptlet("prevent-addEventListener", "click", "popMagic")`